modules/terraform-aws-vpc/README.md

```diff
@@ -47,10 +49,14 @@ In each directory an example `terraform.tfvars.sample` values file is included t
 | <a name="input_vpc_cidr"></a> [vpc\_cidr](#input\_vpc\_cidr) | VPC CIDR Block | `string` | n/a | yes |
 | <a name="input_vpc_name"></a> [vpc\_name](#input\_vpc\_name) | Name of the VPC. | `string` | n/a | yes |
 | <a name="input_cdp_vpc"></a> [cdp\_vpc](#input\_cdp\_vpc) | Flag to indicate if the VPC is for a CDP environment | `bool` | `true` | no |
+| <a name="input_create_vpc"></a> [create\_vpc](#input\_create\_vpc) | Flag to specify if the VPC should be created. Otherwise data sources will be used to lookup details of existing resources. | `bool` | `true` | no |
```

Comment:
Yes, the module is now called from the aws-pre-reqs in all cases, for both new and pre-existing VPC resources.

For pre-existing the ids of the VPC and subnets are passed as inputs (via the `existing_*` variables) and a data source lookup takes place to get information and do a simple validation. This will allow further validation in the future however.